docs/manual/format.md

```diff
@@ -23,17 +23,20 @@ package file is divided in 4 logical sections:
 . Payload   -- compressed archive of the file(s) in the package (aka "payload")
 ```
 
-All 2 and 4 byte "integer" quantities (int16 and int32) are stored in
-network byte order.  When data is presented, the first number is the
-byte number, or address, in hex, followed by the byte values in hex,
-followed by character "translations" (where appropriate).
+All 2 and 4 byte "integer" quantities (int16 and int32) are stored in network
+byte order (big-endian).  When data is presented, the first number is the byte
+number, or address, in hex, followed by the byte values in hex, followed by
+character "translations" (where appropriate).
```

Comment:
I figured it would be OK to increase the line wrapping from 70 characters to 80 (roughly).  Let me know if that's inappropriate, or if it can be wrapped to e.g. 100 characters.

Reply:
I'd not touch the wrapping because it makes it difficult to see the actual content change.